homeassistant/components/fritzbox/__init__.py

```diff
@@ -58,7 +80,7 @@ def ensure_unique_hosts(value):
 )
 
 
-async def async_setup(hass, config):
+async def async_setup(hass: HomeAssistant, config):
     """Set up the AVM Fritz!Box integration."""
     if DOMAIN in config:
```

Comment:
In a future PR we should drop yaml imports.

Reply:
I fully agree :+1: on documentation the yaml way is already not listed anymore, so maybe I'll provide another PR shortly